### PR TITLE
"Add" pinkdragontamer

### PR DESCRIPTION
**Pull request by ghost**
I did get permission for this

### CODE REVIEW

**Comment by HoeenCoder** [comment]
Lets just stick with the name that your set has please.

**Comment by ghost** [comment]
OK :[

File: data/mods/ssb/statuses.js [hunk 1]
```diff
@@ -955,12 +955,6 @@ let BattleStatuses = {
 		noCopy: true,
 		onStart(pokemon) {
 			this.add(`c|+Lost Seso|<3 ( ͡° ͜ʖ ͡°) DANCE WITH ME ( ͡° ͜ʖ ͡°) <3`);
-			if (pokemon.illusion) return;
-			this.effectData.danceMultiplier = 0;
-		},
-		onAfterMove(pokemon, target, move) {
-			if (pokemon.illusion) return;
-			if (move.flags.dance) this.effectData.danceMultiplier++;
```
Comment (HoeenCoder) [inline data/mods/ssb/statuses.js:963]: What about this though?

Comment (ghost) [reply data/mods/ssb/statuses.js:963]: oh yeah. That was code that was never used since Seso got a different move

**Pull request closed without merging**